app/src/layout.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+from panel.layout.flex import FlexBox
+
+from .config_importer import ConfigImporter
+from .plotter import PlotterFactory
+
+
+# Load data
+with open("data/outfile.json", "r") as f:
+    data = json.load(f)
+
+# Load config
+config_importer = ConfigImporter()
+config = config_importer.get_config()
+
+# Initialize plotter factory
+plotter_factory = PlotterFactory()
+
+chart_collection = {}
```

Comment:
It may be nice to add type hinting here, as it is a key component of the code and reimported elsewhere. But up to you, I've noticed you don't use it much and it's also fine

Reply:
Yes, I tend to only use them when writing endpoints as I use them in the tests. But i'd like to introduce this habit in my coding, I will add them here :)